src/reducers/projects.js

```diff
@@ -118,6 +118,14 @@ export default function reduceProjects(stateIn, action) {
         action.meta.timestamp,
       );
 
+    case 'PROJECT_BEAUTIFIED':
+      return state.setIn([action.payload.projectKey, 'sources'],
```

Comment:
`[action.payload.projectKey.sources]` doesn't actually do what we want - i think we still need to hard code 'sources' here, since the structure is `projectkey: { sources: {...}}`